x-pack/plugins/observability_solution/observability_ai_assistant_app/public/hooks/use_conversation.ts

```diff
@@ -121,6 +121,7 @@ export function useConversation({
     service,
     connectorId,
     onConversationUpdate: (event) => {
+      conversation.refresh();
```

Comment:
Does this cause any weirdness in terms of the timeline jumping? I can also test it out locally but wondering if you've seen it.

Reply:
I tested and can't see any difference compared to `main` (both fail to scroll down properly to show the bottom bar of the last message with the feedback buttons + regenerate button).
I'm going to go ahead and merge this!